lib/data/external/AwsClient.js

```diff
@@ -359,26 +359,21 @@ class AwsClient {
             return callback();
         });
     }
-    copyObject(request, sourceBucketName, sourceObjectKey, sourceLocation,
-    log, callback) {
+    copyObject(request, sourceKey, sourceLocation, log, callback) {
```

Comment:
what's the difference between sourceKey and sourceLocation?

Reply:
`sourceKey` is the AWS source object name, Maybe I should call it `awsSourceKey`
`sourceLocation` is the location of the source object

Reply:
I still am not clear on the difference.

Reply:
In this case: `awsSourceKey` === `bucketname/objectname` and `sourceLocation` === `aws-test`

Reply:
locationConstraintName